Makefile

```diff
@@ -93,10 +97,17 @@ pkg/assets/zz_generated_offsets_linux.go pkg/assets/zz_generated_offsets_windows
 	     -prefix embedded-bins/staging/$(zz_os)/ embedded-bins/staging/$(zz_os)/bin
 endif
 
+
 k0s: TARGET_OS = linux
 k0s: pkg/assets/zz_generated_offsets_linux.go
+k0s: BUILD_GO_CGO_ENABLED = 1
+k0s: GOLANG_IMAGE = "k0sbuild.docker-image.k0s"
+k0s: LD_FLAGS = -w -s -extldflags=-static
```

Comment:
This `LD_FLAGS` assignment is going to clobber the large `LD_FLAGS` defined above.

If you want conditional `LD_FLAGS` based on platform, add them conditionally to `BUILD_GO_LDFLAGS_EXTRA`